xml_converter/doc/position/position.md

```diff
@@ -5,8 +5,10 @@ applies_to: ["Icon"]
 xml_fields: ["Position"]
 compatability: [TacO, Burrito, BlishHUD]
 protobuf_field: position
+xml_parent_export: []
+xml_child_export: ["X Position", "Y Position", "Z Position"]
```

Comment:
are `xml_parent_export` and `xml_child_export` the most descriptive names to use here or are there better options that make more sense when reading them in isolation.